api/v1alpha1/bgppeer_webhook.go

```diff
@@ -102,17 +110,40 @@ func (bgpPeer *BGPPeer) validateBGPPeersRouterID(existingBGPPeersList *BGPPeerLi
 	return nil
 }
 
+func (bgpPeer *BGPPeer) validateBGPPeersMyASN(existingBGPPeersList *BGPPeerList, bgpFrrMode bool) *field.Error {
+	myASN := bgpPeer.Spec.MyASN
+	if !bgpFrrMode {
+		return nil
+	}
+	for _, BGPPeer := range existingBGPPeersList.Items {
+		if myASN != BGPPeer.Spec.MyASN {
+			return field.Invalid(field.NewPath("spec").Child("MyASN"), myASN,
+				fmt.Sprintf("Multiple BGP instances not supported myASN %d existing myASN %d",
```

Comment:
I'd rename this to "multiple local ASN not supported in FRR mode" or something like that.